app/src/main/res/xml/rowkeys_turkish_q1.xml

```diff
@@ -0,0 +1,178 @@
+                latin:additionalMoreKeys="="
+                latin:moreKeys="!text/morekeys_r" />
+            <Key
+                latin:keySpec="t"
+                latin:keyHintLabel="["
+                latin:additionalMoreKeys="!text/keyspec_left_square_bracket"
+                latin:moreKeys="!text/morekeys_t" />
+            <Key
+                latin:keySpec="!text/keyspec_y"
+                latin:keyHintLabel="]"
+                latin:additionalMoreKeys="!text/keyspec_right_square_bracket"
+                latin:moreKeys="!text/morekeys_y" />
+            <Key
+                latin:keySpec="u"
+                latin:keyHintLabel="&lt;"
+                latin:additionalMoreKeys="!text/keyspec_less_than"
+                latin:moreKeys="!text/morekeys_u" />
+            <Key
+                latin:keySpec="ı"
+                latin:keyHintLabel="&gt;"
+                latin:additionalMoreKeys="!text/keyspec_greater_than"
+                latin:moreKeys="!text/morekeys_i" />
+            <Key
+                latin:keySpec="o"
+                latin:keyHintLabel="{"
+                latin:additionalMoreKeys="!text/keyspec_left_curly_bracket"
+                latin:moreKeys="!text/morekeys_o" />
+            <Key
+                latin:keySpec="p"
+                latin:keyHintLabel="}"
+                latin:additionalMoreKeys="!text/keyspec_right_curly_bracket" />
+            <Key
+                latin:keySpec="ğ"
+                latin:moreKeys="!text/morekeys_g" />
+            <Key
+                latin:keySpec="ü"
+                latin:moreKeys="!text/morekeys_u" />
+        </case
+            latin:showNumberRow="true">
+            <Key
+                latin:keySpec="!text/keyspec_q"
+                latin:moreKeys="!text/morekeys_q" />
+            <Key
+                latin:keySpec="!text/keyspec_w"
+                latin:moreKeys="!text/morekeys_w" />
+            <Key
+                latin:keySpec="e"
+                latin:moreKeys="!text/morekeys_e" />
+            <Key
+                latin:keySpec="r"
+                latin:moreKeys="!text/morekeys_r" />
+            <Key
+                latin:keySpec="t"
+                latin:moreKeys="!text/morekeys_t" />
+            <Key
+                latin:keySpec="!text/keyspec_y"
+                latin:moreKeys="!text/morekeys_y" />
+            <Key
+                latin:keySpec="u"
+                latin:moreKeys="!text/morekeys_u" />
+            <Key
+                latin:keySpec="ı"
+                latin:moreKeys="!text/morekeys_i" />
+            <Key
+                latin:keySpec="o"
+                latin:moreKeys="!text/morekeys_o" />
+            <Key
+                latin:keySpec="p" />
+            <Key
+                latin:keySpec="ğ"
+                latin:moreKeys="!text/morekeys_g" />
+            <Key
+                latin:keySpec="ü"
+                latin:moreKeys="!text/morekeys_u" />
+        <case>
+        </case>
```

Comment:
Also why empty case tag exists?

Reply:
While I was trying to create cases to be able to show symbols based on related setting, i created more than enough cases and forget to delete this one. I'll delete it and recommit. By the way, I did not build and test this commit only because it's simply xml. Nevertheless it gives errors...

Reply:
I think i resolved it. I wrongly added key tags outside of case tag, hence the empty case tag. Edited and updated.